wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md

```diff
@@ -0,0 +1,56 @@
+
+Los postulantes necesitan cumplir con una actividad de modeo, subir por lo menos dos mapas que hayan modeado recientemente y completar un test en el [Criterio de Rankeo](/wiki/Ranking_Criteria) en orden para poder postular. Hay un periodo de 90 días para poder postular nuevamente una vez que la primera se ha mandado. Un mensaje del NAT le será enviado una vez que se ha decidido el veredicto. Más información acerca de la postulación puede ser encontrada en el sitio web.
+
+## Criterio básico
+
+Los Modders que apuntan a postular a los Beatmap Nominators deben de cumplir con el siguiente criterio:
+
+- Han estado modeando mapas activamente por lo menos durante los pasados 3 meses antes de postular.
+- Han acumulado por lo menos 150 o 200 [kudosu](/wiki/Glossary#kudosu!) en total, dependiendo del modo al que aplicarán.
+- Han proporcionado de 2 a 4 de sus modeos, los que, según ellos, creen que los pueda calificar para ser parte de los Beatmap Nominators.
+
+### Qué califica como un post de modeo
+
+- Al registrarse, el sitio web calculará automáticamente el puntaje kudosu de los últimos 3 meses.
+- Un post de modeo calificado es un post de un usuario en la discusión sobre un beatmap, en donde por lo menos una sugerencia/problema fue premiado con un voto, y ha sido creado dentro de los últimos 3 meses.
+- El modo de juego que se modee no importa, ya que la actividad en la comunidad de modeo es medida como un todo y no como la actividad en cierto modo de juego.
+- El punto en el tiempo en el que se te premia con kudosu por un post de modeo **sí** importa, ya que la calculadora incluida en el sitio web buscará los datos desde tu Historial de eventos de Modeo.
+
+### Expectativas de modeo
+
+Los modeos subidos deben de mostrar que el usuario es capaz de analizar y mejorar los mapas a un ajuste estándar para los Beatmap Nominators.
+
+Los siguientes son características de modding esperados de los Beatmap Nominators. Fallar en mostrar cualquiera de estas características puede resultar en una postulación rechazada.
+
+- Cubrir todas los niveles de dificultad
+- Identificación de problemas inrankeables, incluyendo los que las herramientas no pueden detectar, tales como tiempos incorrectos o metadatos
+- Comparación entre partes del mapa para problemas de soporte o sugerencias
+- Comentar acerca de la gran variedad de elementos de un mapa, como el ritmo, espaciado, movimiento, intensidad, contraste y consistencia
+- Identificación de problemas aislados como de temática general
+
+### Cómo el puntaje mínimo es calculadora
+
+- Esto depende del total de `Puntaje de Modeo` que el postulante ha alcanzado durante los últimos meses. Este valor se determina de acuerdo a  la siguiente fórmula:
+
+![Fórmula del Puntaje de Modeo](/wiki/shared/Modscore_new_wiki.png "Fórmula del Puntaje de Modeo")
+
+`m` = Número mínimo esperado de beatmaps modeados individualmente en un mes
+`M` = Número total de beatmaps modeados individualmente en un mes
```

Comment:
I don't know why, but on rich diff it shows that it's not like this...

Reply:
Line 40 in the English version has a line break (indicated by the 2 trailing spaces on line 40). For your PR, please use a trailing backslash instead.

```suggestion
`m` = Número mínimo esperado de beatmaps modeados individualmente en un mes\
`M` = Número total de beatmaps modeados individualmente en un mes
```